addons/website/views/snippets.xml

```diff
@@ -1056,6 +1070,54 @@
         </we-select>
     </div>
 
+    <!-- DISPLAY | .s_countdown -->
+    <div data-js="countdown" data-selector=".s_countdown">
+        <we-datetimepicker class="o_we_full_width" string="Due Date" data-select-data-attribute="" data-attribute-name="endTime"/>
+        <we-row>
+            <we-select string="What should happen at the end?" data-no-preview="true">
+                <we-button data-end-action="nothing">Nothing</we-button>
+                <we-button data-end-action="redirect">Redirect</we-button>
+                <we-button data-end-action="message">Show Message</we-button>
+            </we-select>
+            <we-button class="far fa-edit fa-fw align-self-end toggle-edit-message" title="Edit Message"/>
+        </we-row>
+        <we-checkbox string="Hide countdown at the end" data-select-class="hide-countdown"/>
+        <we-urlpicker class="o_we_full_width" string="URL" data-select-data-attribute="" data-attribute-name="redirectUrl" data-default-value="/my-awesome-page"/>
```

Comment:
data-default-value does not exists anymore?

Reply:
No, put it in data-select-data-attribute